examples/gallery/images/rgb_image.py

```diff
@@ -0,0 +1,39 @@
+"""
+RGB Image
+---------
+The :meth:`pygmt.Figure.grdimage` method can be used to plot Red, Green, Blue
+(RGB) images, or any 3-band false color combination. Here, we'll use
+:meth:`rioxarray.open_rasterio` to read a GeoTIFF file into an
+:class:`xarray.DataArray` format, and plot it on a map.
+
+The example below shows a Worldview 2 satellite image over
+`Lāhainā, Hawaiʻi during the August 2023 wildfires
+<https://en.wikipedia.org/wiki/2023_Hawaii_wildfires#L%C4%81hain%C4%81>`_.
+Data is sourced from a Cloud-Optimized GeoTIFF (COG) file hosted on
+`OpenAerialMap <https://map.openaerialmap.org>`_ under a
+`CC BY-NC 4.0 <https://creativecommons.org/licenses/by-nc/4.0/>`_ license.
+"""
+import pygmt
+import rioxarray
+
+###############################################################################
+# Read 3-band data from GeoTIFF into an xarray.DataArray object
+image = rioxarray.open_rasterio(
+    filename="https://oin-hotosm.s3.us-east-1.amazonaws.com/64d6a49a19cb3a000147a65b/0/64d6a49a19cb3a000147a65c.tif",
+    overview_level=5,
+)
+# Subset to area of Lāhainā in EPSG:32604 coordinates
+image = image.rio.clip_box(minx=738000, maxx=755000, miny=2300000, maxy=2318000)
+image
+
+###############################################################################
+# Plot the RGB imagery
+fig = pygmt.Figure()
+with pygmt.config(FONT_TITLE="Times-Roman"):  # Set title font to Times-Roman
+    fig.grdimage(
+        grid=image,
+        projection="x1:100000",
+        frame=[r"WSne+tL@!a\225hain@!a\225, Hawai\140i on 9 Aug 2023", "af"],
```

Comment:
Changed to using `\140` instead of `\047` as it looks a bit closer to the [Okina](https://en.wikipedia.org/wiki/%CA%BBOkina) `ʻ` letter (U+02BB) (see also https://github.com/GenericMappingTools/pygmt/pull/1474#discussion_r1306286723). Now it looks like this:

![Lāhainā, Hawaiʻi in Times-Roman font.](https://github.com/GenericMappingTools/pygmt/assets/23487320/a71def73-2826-4dd4-9cee-e7ed7a8f9ce6)

If anyone has ideas on how to use the actual Okina letter, let me know! I did try using `ʻ` directly, but it shows up like this:

![Hawaiʻi with the Okina letter plotted incorrectly.](https://github.com/GenericMappingTools/pygmt/assets/23487320/18356d72-b3f1-4ae4-b9a7-bf974420ca81)